src/domain/collaboration/callout/graphql/CalloutQueries.graphql

```diff
@@ -189,6 +189,7 @@ fragment CalloutCardTemplate on Callout {
 fragment CalloutCanvasTemplate on Callout {
   canvasTemplate {
     id
+    value
```

Comment:
have you checked that this doesn't have a lot of overhead? 
I'm not sure, but I thought we removed the value on purpose and added the fragment CanvasTemplateWithValue, is this needed?